### PR TITLE
 Enable session and speaker creation check

### LINKED ISSUE #924: Show appropriate message if session-speakers arent enabled
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
Currently even if there are no tracks present the create session page i.e events/event_id/sessions/create route enables user to submit proposal something like:
create-session

**Expected behavior:**
A message should be shown that "There are no tracks available".

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #924 
create-session-fix

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-925-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@geekyd Please review.

File: app/templates/events/view/sessions/create.hbs [hunk 1]
```diff
@@ -1,9 +1,13 @@
 <div class="column sixteen wide">
-  <h2 class="ui header center aligned">
-    {{t 'Create Session'}}
-  </h2>
-  <div class="ui container">
-    {{forms/session-speaker-form fields=model.form data=model save=(action 'save')
-      includeSpeaker=true includeSession=true isSessionSpeaker=true isLoading=isLoading}}
-  </div>
+  {{#if model.tracks}}
```
Comment (geekyd) [inline app/templates/events/view/sessions/create.hbs]: Use `isSessionsSpeakersEnabled`
Implement something like this for both speaker & session pages
screen shot 2017-10-15 at 9 47 18 pm

**Comment by niranjan94** [comment]
Deployment link: https://pr-925-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@sumedh123 Do the same for speakers page.

**Comment by sumedh123** [comment]
@geekyd The server has the implementation that I followed. Should we do it in list anyways then?

**Comment by geekyd** [comment]
@sumedh123 Yeah the server also follows the similar behaviour, we should not show the table and the create button if the sessions & speakers are not enabled.

**Comment by sumedh123** [comment]
@geekyd I did it for sessions for now. We can do it for speakers in #646 since doing it now will anyways show speakers?

**Comment by niranjan94** [comment]
Deployment link: https://pr-925-open-event-frontend-fossasia.surge.sh

File: app/templates/events/view/sessions.hbs [hunk 2]
```diff
@@ -1,51 +1,58 @@
...
+      {{outlet}}
     </div>
-  {{/if}}
-  <div class="row">
-    {{outlet}}
   </div>
-</div>
+{{else}}
+  <div class="sixteen wide column">
+    <h2 class="centered text">{{t 'In order to use this feature please activate "Sessions and Speakers" for the event.'}}</h2>
```
Comment (geekyd) [inline app/templates/events/view/sessions.hbs]: Lets use `h3` here. We are using `h2` for displaying the name of the event.

File: app/templates/events/view/sessions.hbs [hunk 3]
```diff
@@ -1,51 +1,58 @@
...
+    <p class="centered text">{{t 'Please click '}}<a href="{{href-to 'events.view.edit.sessions-speakers'}}">here</a>{{t ' to activate this module.'}}</p>
```
Comment (geekyd) [inline app/templates/events/view/sessions.hbs]: Use `h4` or `h3` here the text is too small

**Comment by sumedh123** [comment]
@geekyd Done. :thumbsup:

**Comment by niranjan94** [comment]
Deployment link: https://pr-925-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@CosmicCoder96 Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-925-open-event-frontend-fossasia.surge.sh

**Pull request merged**